src/supremm/assets/mongo_setup.js

```diff
@@ -440,5 +440,11 @@ var summarydef = {
 };
 
 db = db.getSiblingDB("supremm");
-db.schema.update({_id: sdef._id}, sdef, {upsert: true});
-db.schema.update({_id: summarydef._id}, summarydef, {upsert: true});
+var version = parseFloat(db.version());
+if (version < 5) {
+    db.schema.update({_id: sdef._id}, sdef, {upsert: true});
+    db.schema.update({_id: summarydef._id}, summarydef, {upsert: true});
+} else {
+    db.schema.updateOne({_id: summarydef._id}, {$set: summarydef}, {upsert: true});
```

Comment:
Should this be `sdef` rather than `summarydef`?

Reply:
Yes that should... my mistake

Reply:
Just fixed that typo and tested it out again.